fah_visualizer.py

```diff
@@ -13,14 +13,14 @@
     preload_top, preload_tops, save_trajs, save_generic, backup, load_trajs
 from multiprocessing import Pool
 
-# STEP 0.5 META pickle file of trajectories
-atom_index_start, atom_index_end, number_between =\
-    [int(x) for x in input("What is the start, end, and jump between atom index numbers: ").split()]
+# STEP 0.5 required inputs for script
 traj_name = input("Name of the trajectory files: ")  # Note need to be in 'trajectory-{run}.xtc' format for function
```

Comment:
nice use of "input" but the hard coding of `trajectory-{run}.xtc` may not be compatible with every lab (including ours. Maybe have it search for `*.xtc` files or something?